themes/tailwind/assets/index.css

```
@@ -0,0 +1,245 @@
+    --vc-color-info-900: var(--color-sky-900);
+    --vc-color-info-950: var(--color-sky-950);
+}
+
+/* -----------------------------------------------------------------
+   Tailwind v4 @theme block — expose vuecs tokens as Tailwind
+   color/radius names so components can write e.g. `bg-primary-600`,
+   `text-fg`, `rounded-md` and have them resolve to the vc token
+   chain. Non-inline so browser-time swaps (.dark, setColorPalette) work.
+   ----------------------------------------------------------------- */
+
+@theme {
```

Stylelint (17.10.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)